### PR TITLE
add index.html and index.css

### PR DESCRIPTION
**Pull request by simonlj8**
Added som DIV's to index in startpage

### CODE REVIEW

**Comment by matodorovic** [comment]
Looks good.

**Review by matodorovic — APPROVED** [review]
👍

**Review by mhema42 — APPROVED** [review]

**Comment by mhema42** [comment]
Looks good

File: index.html [hunk 1]
```diff
@@ -1,14 +1,26 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <link rel="stylesheet" type="text/css" href="main.css">
+        <link rel="stylesheet" type="text/css" href="src/scss/index.css">
```
Comment (matodorovic) [inline index.html:4]: Ta bort src/scss

**Review by matodorovic — CHANGES_REQUESTED** [review]

**Pull request closed without merging**